identity/client/src/pages/setup/SetupPage.tsx

```diff
@@ -69,7 +72,8 @@ const SetupForm: React.FC<SetupFormProps> = ({ onSuccess }) => {
       if (success) {
         onSuccess();
       } else {
-        setSubmitError(error?.detail || "");
+        const detail = (error as ErrorResponse<"detailed">)?.detail;
+        setSubmitError(detail || "");
```

Comment:
@vitorwtavares I think it's better to provide some generic error message instead of having an empty string here